woocommerce/woocommerce
# Remove insight first product and payment note

### All Submissions:






### Changes proposed in this Pull Request:

Remove the insight first product and payment note.

Part of #34930






### How to test the changes in this Pull Request:



1. Create a site between 1 day and 1 week old (or manually adjust the `woocommerce_admin_install_timestamp` option to this timeframe)
2. Install a plugin like [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)
3. Under Tools -> Cron Events, run `wc_admin_daily`
4. Make sure the note with a title of "Insight" has not been added



### Other information:




### FOR PR REVIEWER ONLY:
